omod-2.2/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/PatientProgramAttributeResource2_2Test.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_2;
+
+import org.openmrs.PatientProgramAttribute;
+import org.openmrs.api.ProgramWorkflowService;
+import org.junit.Before;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.web.v1_0.RestTestConstants2_2;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResourceTest;
+import org.springframework.beans.factory.annotation.Autowired;
+
+//@org.springframework.test.context.ContextConfiguration(locations = { "classpath:TestingApplicationContext.xml" }, inheritLocations = true)
```

Comment:
Do you need the above commented out line?

Reply:
I'm not sure, I'm still debugging trying to find the cause of the error

Reply:
The only error that i have with your pushed changes is the one i mentioned on talk. And it should be an obvious one to fix. Unless there are local changes that you have not yet pushed.